root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -281,6 +281,49 @@ const testData = [
     expected_relationship_type: 'downloadpurchase',
             expected_clean_url: 'https://fr-ca.7digital.com/artist/83-1/release/récidivistes-12888712',
   },
+  // ACUM
+  {
+          input_url: 'https://nocs.acum.org.il/acumsitesearchdb/album?albumid=000608',
+          input_entity_type: 'release',
+          expected_relationship_type: 'otherdatabases',
+          expected_clean_url: 'https://nocs.acum.org.il/acumsitesearchdb/album?albumid=000608',
+          only_valid_entity_types: ['release'],
+  },
+  {
+          input_url: 'https://nocs.acum.org.il/acumsitesearchdb/work?workid=29AJOVQ&tab=musical',
+          input_entity_type: 'work',
+          expected_relationship_type: 'otherdatabases',
+          expected_clean_url: 'https://nocs.acum.org.il/acumsitesearchdb/work?workid=29AJOVQ',
+          only_valid_entity_types: ['work'],
+  },
+  {
+          input_url: 'https://nocs.acum.org.il/acumsitesearchdb/version?workid=1031256&versionid=1031256001',
+          input_entity_type: 'work',
+          expected_relationship_type: 'otherdatabases',
+          expected_clean_url: 'https://nocs.acum.org.il/acumsitesearchdb/work?workid=1031256',
+          only_valid_entity_types: ['work'],
+  },
+  {
+          input_url: 'https://nocs.acum.org.il/acumsitesearchdb/results?creatorid=I-000137240-1',
+          input_entity_type: 'label',
+          expected_relationship_type: 'otherdatabases',
+          expected_clean_url: 'https://nocs.acum.org.il/acumsitesearchdb/results?creatorid=I-000137240-1',
+          only_valid_entity_types: ['artist', 'label'],
+  },
+  {
+          input_url: 'https://nocs.acum.org.il/acumsitesearchdb/results?creatoripnnumber=00551105395',
```

Comment:
I'm not sure I see the point of storing these IPI search results in particular, since we already store IPI numbers. So you can generate such links automatically.

There are other IDs here which we can't store yet, e.g. "I-000137240-1" (which looks like an "IPI base number" to me?). I think it's fine to link to these results until custom attributes are supported, though it's technically a misuse of the "other databases" relationship type, since it's not a real page for the entity.

I'm just talking about the `/results` links BTW. These appear to be actual pages for the entity:
https://nocs.acum.org.il/acumsitesearchdb/album?albumid=000608
https://nocs.acum.org.il/acumsitesearchdb/work?workid=29AJOVQ
https://nocs.acum.org.il/acumsitesearchdb/version?workid=1031256&versionid=1031256001

Reply:
Those ACUM identifiers (e.g. "I-000137240-1") are the main motivation for this as my user script [acum-work-import](https://github.com/dvirtz/musicbrainz-scripts/blob/main/src/acum-work-import/README.md) import works from ACUM and matching the writers using IPI if there's one and otherwise by name/alias which might give false matches if there's more than one match, so by linking the ACUM creator page I can have one to one mapping for all artists.

I'd be happy to hear other suggestions to achieve this.

Reply:
I think the point was never not to match the ACUM IDs, but whether `creatoripnnumber` is just the IPI vs `creatorid` being the ACUM ID? In which case we could still link to the ACUM IDs but just store IPIs in the normal way we already have :)

Reply:
Thanks for the clarification @reosarevok, I seem to misread @mwiencek comments.
Removed support for creatoripnnumber links.